sdk/monitor/monitor-ingestion/src/logsIngestionClient.ts

```diff
@@ -9,14 +9,21 @@ import { GZippingPolicy } from "./gZippingPolicy";
 import { concurrentRun } from "./utils/concurrentPoolHelper";
 import { splitDataToChunks } from "./utils/splitDataToChunksHelper";
 import { isError } from "@azure/core-util";
+import { KnownMonitorAudience } from "./constants";
 /**
  * Options for Monitor Logs Ingestion Client
  */
 export interface LogsIngestionClientOptions extends CommonClientOptions {
   /** Api Version */
   apiVersion?: string;
+
+  /**
+   * The Audience to use for authentication with Azure Active Directory (AAD). The
```

Comment:
Use the new product name
```suggestion
   * The Audience to use for authentication with Microsoft Entra ID. The
```